### PR TITLE
 CASSANDRA-19003  Get sidecar port through cassandra context

### PR DESCRIPTION
**Pull request by sarankk**

### CODE REVIEW

File: cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraContext.java [hunk 1]
```diff
@@ -96,6 +96,11 @@ public SidecarClient getSidecarClient()
         return sidecarClient;
     }
 
+    public int getSidecarPort()
```
Comment (frankgh) [inline cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraContext.java]: NIT:
```suggestion
    public int sidecarPort()
```

**Review by frankgh — APPROVED** [review]
One small NIT, but looks good to me!

**Review by dineshjoshi — APPROVED** [review]

**Review by dineshjoshi — APPROVED** [review]
+1

**Comment by frankgh** [comment]
Closed via https://github.com/apache/cassandra-analytics/commit/87a729feb4660f57bacb2a4be73e1bb2d509578b

**Pull request closed without merging**